jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>I honestly hate this last part because no one is gonna read this except people who skimmed the whole thing and then skipped to the end, in which case the end should be a summary NOT the caveats which it currently is -_-</p><p> </p><p>And wtf is this quiz? I mean sure</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='151'/>